### DISCUSSION TITLE
CKEditor plugin

### DISCUSSION THREAD

**Discussion by soke1556** [requarks/wiki#3385] [Help / Questions]
**I need help adding CKEditor5 capabilities to format code. I cannot use MD, since the docs have been created in RichText. Can it be done without being affected by future wiki.js updates?**

OS: Linux - Ubuntu (bionic) 18.04 x64
Wiki.js version: 2.3.81
Database engine: postgres 10.12
Node.js: 12.13.1

**Comment by canuckbrian** [comment]
I'm also looking for how to do this.

I want to be able to enable the Codeblocks Plugin for CKEditor 5. Where can I do this?
